tests/test_integration.py

```diff
@@ -8,22 +8,22 @@
 import yaml
 
 file_dir_path = os.path.dirname(__file__)
-TEST_ITEMS = os.path.join(file_dir_path,"test_items.yaml")
+TEST_ITEMS = os.path.join(file_dir_path, "test_items.yaml")
 
 class TestGet(object):
     def setUp(self):
         """Server Settings"""
-        self.url = 'http://localhost:8889'
+        self.url = 'http://localhost:9761'
         self.api = requests.get(self.url + '/api/v1')
-        assert_true(self.api.ok)
+        #assert_true(self.api.ok)
```

Comment:
Should this be kept?

Reply:
Good catch. Should probably add that back into action!